### PR TITLE
Cr/27/google

### PR DESCRIPTION
**Pull request by carin91**

### CODE REVIEW

File: src/carin/A27/imgs/search.png:Zone.Identifier
Comment (wfercanas) [inline src/carin/A27/imgs/search.png:Zone.Identifier:1]: Este archivo se puede eliminar

File: src/carin/A27/index-a27.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            <button type="submit">Buscar con Google</button></a
+          >
+
+          <a href="https://doodles.google/">
+            <button type="submit">Voy a tener suerte</button></a
+          >
+        </div>
+      </form>
+    </main>
+    <div id="sub-form"></div>
```
Comment (wfercanas) [inline src/carin/A27/index-a27.html:57]: Este `div` vacío sobra

File: src/carin/A27/index-a27.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        <div id="search">
+          <a href="google.com"
+            ><img id="search-img" src="imgs/search.png" alt="google search"
+          /></a>
+          <input type="search" id="site-search" name="google-search" />
+          <a href="google.com"><img src="imgs/mic.png" alt="google mic" /></a>
+          <a href="google.com"
+            ><img id="cam-img" src="imgs/camera.png" alt="google cam"
+          /></a>
+        </div>
```
Comment (wfercanas) [inline src/carin/A27/index-a27.html:45]: La barra de búsqueda se está desbordando en responsive:
image

File: src/carin/A27/index-a27.html [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+          <ul>
+            <a href="https://policies.google.com/privacy?hl=es-419&fg=1">
+              Privacidad</a
+            >
+            <a href="https://policies.google.com/terms?hl=es-419&fg=1">
+              Condiciones</a
+            >
+            <a href="google.com"> Preferencias</a>
+          </ul>
+        </div>
```
Comment (wfercanas) [inline src/carin/A27/index-a27.html:95]: Cuando ya no cabe la lista, está se está colocando abajo de forma correcta, pero por fuera del contenedor padre:
image

**Review by wfercanas — CHANGES_REQUESTED** [review]

File: src/carin/A27/index-a27.html (hunk 1 above)
Comment (carin91) [reply src/carin/A27/index-a27.html:57]: attended
It was included to (1) add to the page additional content for festivities and so on (2) to organize the sections accordingly to google.com, so, it was necessary to change display: flex to display: grid in body. It was also necessary to declare the row lines for the footer.

image

image

image

File: src/carin/A27/index-a27.html (hunk 2 above)
Comment (carin91) [reply src/carin/A27/index-a27.html:45]: 1) In device emulation is not overlaying

image

2) In normal mode the behavior of google.com is similar, with the search bar overlaying

image

3) I fix the width declaration for the search, i had 2 width values (50vw and 400px), so i deleted the 400px line.
image

File: src/carin/A27/index-a27.html (hunk 3 above)
Comment (carin91) [reply src/carin/A27/index-a27.html:95]: Attended
I had a height value of 50px, i changed it to auto and add 50px value to min-height property.
image

File: src/carin/A27/index-a27.html
Comment (wfercanas) [inline src/carin/A27/index-a27.html:1]: El documento está utilizando más de 100vh y está apareciendo un scroll.

**Review by wfercanas — CHANGES_REQUESTED** [review]

File: src/carin/A27/index-a27.html
Comment (carin91) [reply src/carin/A27/index-a27.html:1]: attended
I changed the body height property from 100% to 100vh.
image

File: src/carin/A27/imgs/search.png:Zone.Identifier
Comment (carin91) [reply src/carin/A27/imgs/search.png:Zone.Identifier:1]: attended
image

**Review by wfercanas — APPROVED** [review]

**Pull request merged**